### ISSUE TITLE
DEADLINE_EXCEEDED on Ava Engine CPU

### ISSUE THREAD

**Issue by blechatellier** [ImageIntelligence/imageintelligence#14]
When running `node classification.js` the first time grpc throws an error. All subsequent calls work as expected. Using 0.70 CPU on macOS 10.13.2 with nodeJS client.

```
{ Error: 4 DEADLINE_EXCEEDED: Deadline Exceeded
    at Object.exports.createStatusError (/Users/brice/Desktop/ava/node_modules/grpc/src/common.js:87:15)
    at Object.onReceiveStatus (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:1188:28)
    at InterceptingListener._callNext (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:564:42)
    at InterceptingListener.onReceiveStatus (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:614:8)
    at callback (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:841:24)
  cause:
   { Error: 4 DEADLINE_EXCEEDED: Deadline Exceeded
    at Object.exports.createStatusError (/Users/brice/Desktop/ava/node_modules/grpc/src/common.js:87:15)
    at Object.onReceiveStatus (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:1188:28)
    at InterceptingListener._callNext (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:564:42)
    at InterceptingListener.onReceiveStatus (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:614:8)
    at callback (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:841:24)
     code: 4,
     metadata: Metadata { _internal_repr: {} },
     details: 'Deadline Exceeded' },
  isOperational: true,
  code: 4,
  metadata: Metadata { _internal_repr: {} },
  details: 'Deadline Exceeded' }
```

**Comment by blechatellier** [comment]
Having a similiar issue with faceRecognition

```
➜  examples node faceRecognition.js
Initializing AvaEngine with global features we want enabled...
Initializing AvaEngine with "faceRecognition"...
Unhandled rejection Error: 4 DEADLINE_EXCEEDED: Deadline Exceeded
    at Object.exports.createStatusError (/Users/brice/Desktop/ava/node_modules/grpc/src/common.js:87:15)
    at Object.onReceiveStatus (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:1188:28)
    at InterceptingListener._callNext (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:564:42)
    at InterceptingListener.onReceiveStatus (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:614:8)
    at callback (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:841:24)

➜  examples node faceRecognition.js
Initializing AvaEngine with global features we want enabled...
Skipping initializing, face recognition already initialized.
Retrieving feed for the given feedId=1NJdOI_1
Configuring feed Feeds.Configure feedId=1NJdOI_1
Performing FaceRecognition.Recognize on 3 images...
Unhandled rejection Error: 13 INTERNAL: ClassificationError Models are not loaded
    at Object.exports.createStatusError (/Users/brice/Desktop/ava/node_modules/grpc/src/common.js:87:15)
    at Object.onReceiveStatus (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:1188:28)
    at InterceptingListener._callNext (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:564:42)
    at InterceptingListener.onReceiveStatus (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:614:8)
    at callback (/Users/brice/Desktop/ava/node_modules/grpc/src/client_interceptors.js:841:24)
```

**Comment by dbousamra** [comment]
Fixed in 0.7.1
